CONTRIBUTING.md

```diff
@@ -1,106 +1,186 @@
+4. or run test against **service.proj** in the repo root, passing the directory name of the specific service as a property. e.g. `dotnet test eng\service.proj /p:ServiceDirectory=eventhub --filter TestCategory!=Live`
+
+### Single Service from Visual Studio
+
+1. Build.
+2. Test Explorer window will get populated with tests. Select test and `Run` or `Debug`
+
+### All Client Services from Command Line
+
+1. Open VS 2019 Command Propmpt
+2. Navigate to repository root directory
+3. Invoke `dotnet test eng\service.proj --filter TestCategory!=Live`
+   <br/><br/>
+
+# Management Libraries
+## TO BUILD:
+
+### If you are building from Visual Studio, add a nuget feed source that points to < root >\tools\LocalNugetFeed directory
+
+1.  Open any solution, eg "SDKs\Compute\Microsoft.Azure.Management.Compute.sln"
+2.  Build solution from Visual Studio
+
+### Full Build from command line
+
+1.  Open VS 2019 command prompt
+2.  Navigate to repository root directory
+3.  Invoke `msbuild build.proj /t:Build`
+
+### Single Service from Command Line
+
+1. Open VS 2019 command Prompt
+2. From the root directory
+3. Invoke `msbuild build.proj /t:CreateNugetPackage /p:scope=Compute`
 
-### To run tests:
-Using Visual Studio:
-  - Build project in VS
-  - "Test Explorer" window will get populated with tests. Right click on a test that you wish to Run/Debug.
+> _Build_ without any scope will build all management SDK's.
 
-Using the command line:
-msbuild .\build.proj /t:Runtests /p:Scope=Compute
-in the above example RunTests will build and run tests for Compute only
-or you can use command line CLI
-dotnet test Compute\Microsoft.Azure.Management.Compute\tests\Microsoft.Azure.Management.Tests.csproj
+### Create single nuget package
 
-#### Create single nuget package
 In order to build one package and run it's test
 `msbuild build.proj /t:CreateNugetPackage /p:scope=Compute`
 Nuget package will be created in root directory under \artifacts\packages\Debug (default configuration is Debug)
 
+## TO TEST:
+
+### Using Visual Studio:
+
+1. Build project in Visual Studio.
+2. **Test Explorer** window will get populated with tests. Select test and `Run` or `Debug`
+
+### Using the command line:
+
+Run e.g. `msbuild .\build.proj /t:"Runtests" /p:Scope=Compute`
+In the above example _RunTests_ will build and run tests for Compute only or you can use command line CLI
+`dotnet test Compute\Microsoft.Azure.Management.Compute\tests\Microsoft.Azure.Management.Tests.csproj`
+
 ### Non-Windows command line build
-Now you can use the same command on non-windows as above
-for e.g. on Ubuntu you can do something like below:
-`dotnet msbuild build.proj /t:Build /p:scope=Compute`
-`dotnet msbuild build.proj /t:RunTests /p:scope=Compute`
-`dotnet msbuild build.proj /t:CreateNugetPackage /p:scope=Compute`
+
+Now you can use the same command on non-windows as above for e.g. on Ubuntu you can do something like below:
+
+- `dotnet msbuild build.proj /t:Build /p:scope=Compute`
```

Comment:
@shahabhijeet my goal is that we don't need to use msbuild and these targets manually but instead we can simply call `dotnet build`, `dotnet test`, and `dotnet pack` which are the default .NET cli commands. Behind the scenes we can still call those targets but people won't need to know these custom command lines and targets.